src/hooks/useLogin.ts

```diff
@@ -85,7 +85,7 @@ export const useLogin = () => {
             ({
                 openRenown,
                 user,
-                status,
+                status: user ? 'authorized' : status,
```

Comment:
We should also move the user login stuff into a context. It's one of the cases where managing state inside React is just making things more difficult.